### PR TITLE
Add some Eunomia tests

### PR DESCRIPTION
**Pull request by ablack3**

### CODE REVIEW

**Comment by ablack3** [comment]
@anthonysena Will you review this PR for me?

**Comment by anthonysena** [comment]
Yes, sorry for the long delay here @ablack3. I plan to take a look at this next week. Thanks!

**Review by anthonysena — COMMENTED** [review]
Thanks for these changes and things look good from my review. I did have one question before merging. The newly added unit tests (i.e. the Aggregation test) appears to only use Eunomia for testing which I presume is by design since it is not use SQL and we would not need to test it explicitly against the other RDBMS. Is that the thinking there? Just wanted to confirm.

File: DESCRIPTION [hunk 1]
```diff
@@ -31,7 +31,8 @@ Imports:
   readr,
   rlang,
   RSQLite,
-  DBI
+  DBI,
+  Eunomia
```
[the remark this answers is not in the document]
Comment (ablack3) [reply DESCRIPTION:35]: Yes it should be added to "Suggests". Good catch.

**Comment by ablack3** [comment]
> Thanks for these changes and things look good from my review. I did have one question before merging. The newly added unit tests (i.e. the Aggregation test) appears to only use Eunomia for testing which I presume is by design since it is not use SQL and we would not need to test it explicitly against the other RDBMS. Is that the thinking there? Just wanted to confirm.

Yes these tests only rely on Eunomia and R so they can be executed in any environment including on CRAN servers. I'm basically trying to help get the coverage up over 80% and using Eunomia seems like a good approach. I have not been able to run tests on the OHDSI databases and probably need a tutorial on how to set that up at some point. I wanted to see if you liked what I've done so far before adding more Eunomia tests. If this looks good then I can add more tests when I get a chance. _To 80% and beyond!!_

File: DESCRIPTION (hunk 1 above)
[the remark this answers is not in the document]
Comment (anthonysena) [reply DESCRIPTION:35]: I'll change this after merging - everything else looks good. Thanks for these additions!

**Pull request merged**